uber/models.py

```diff
@@ -1048,7 +1049,10 @@ def _status_adjustments(self):
     @presave_adjustment
     def _staffing_adjustments(self):
         if self.ribbon == c.DEPT_HEAD_RIBBON:
-            self.staffing = self.trusted = True
+            self.staffing = True
+            # question: do we have info about what this person is a dept head of? or is that not something
+            # we know explicitly?  if so, we can either make them trusted in every dept, or trusted in no departments.
+            # self.trusted_depts = self.assigned_depts # this might be a reasonable thing to do. or not.
```

Comment:
Isn't the point of this change that "trusted" doesn't mean the same thing in each department? If that is true, you need to know, for each person, which department trusts them. IMO that should include DHs. So however you're going to mark Staff as trusted in one or more departments, you do the same for Staff-that-have-DH-ribbon. E.g. for each defined department include a list containing at least one staff that is/are considered the DH for that department.